README.md

```diff
@@ -45,5 +45,25 @@ gulp test/dotnet
 2. Next, implement the ```run``` method under the rule that actually does the validation. Add a reference to this script file under ```src/typescript/azure-openapi-validator/index.ts```.
 3. Lastly add a test case for the validation rule, by adding a test json/yaml under ```src/typescript/azure-openapi-validator/tests/resources``` and a script under ```src/typescript/azure-openapi-validator/tests``` depending on the type of the validation rule.
 
+## Changelog
+### What's New (07/07/2017)
+Resolved issues

```

Comment:
These 2 seems to be bug fixes as some of the one below, is there are reason why they are split? We should try following same format, a sentence and a link?